dds/DCPS/FilterEvaluator.cpp

```diff
@@ -125,11 +125,22 @@ FilterEvaluator::SerializedForEval::lookup(const char* field) const
     return iter->second;
   }
   Message_Block_Ptr mb (serialized_->duplicate());
-  // TODO(iguessthislldo): Use Encap
   Serializer ser(mb.get(),
     cdr_ ? Encoding::KIND_XCDR1 : Encoding::KIND_UNALIGNED_CDR, swap_);
```

Comment:
```suggestion
  Serializer ser(mb.get(), Encoding::KIND_UNALIGNED_CDR, swap_);
```
EncapsulationHeader is encoding agnostic, so we can let it default to unaligned.